creator-node/src/components/tracks/tracksComponentService.ts

```diff
@@ -20,7 +20,15 @@ const TrackContentUploadManager = require('./trackContentUploadManager')
  * upload track segment files and make avail - will later be associated with Audius track
  * @dev - Prune upload artifacts after successful and failed uploads. Make call without awaiting, and let async queue clean up.
  */
-const handleTrackContentRoute = async ({ logContext }, requestProps) => {
+export const handleTrackContentRoute = async (
+  { logContext }: { logContext: { requestId: string } },
```

Comment:
im pretty sure this will have other fields other than requestId. should we update this to type any?

Reply:
I'd avoid using `any` as much as possible. It's fine if it has other fields, by adding `{ logContext: { requestId: string } }` we're saying it should _at least_ have a `logContext` field and that should _at least_ have a `requestId` field. Whether it has more won't cause a problem with the typechecker